src/hooks/useActionCenterSubscriber/checkStatusHandlers/getTradeStatusHandler.ts

```diff
@@ -0,0 +1,275 @@
+                },
+              }),
+            )
+
+            const notificationTitle = translate(
+              'notificationCenter.notificationsTitles.swap.title',
+              {
+                sellAmountAndSymbol: numberToCrypto(
+                  fromBaseUnit(swap.metadata.sellAmountCryptoBaseUnit, tradeSellAsset.precision),
+                  tradeSellAsset.symbol,
+                  localeParts,
+                  {
+                    maximumFractionDigits: 8,
+                    omitDecimalTrailingZeros: true,
+                    abbreviated: true,
+                    truncateLargeNumbers: true,
+                  },
+                ),
+                buyAmountAndSymbol: numberToCrypto(
+                  fromBaseUnit(receiveTransfer.value, tradeBuyAsset.precision),
+                  tradeBuyAsset.symbol,
+                  localeParts,
+                  {
+                    maximumFractionDigits: 8,
+                    omitDecimalTrailingZeros: true,
+                    abbreviated: true,
+                    truncateLargeNumbers: true,
+                  },
+                ),
+              },
+            )
+
+            store.dispatch(
+              actionCenterSlice.actions.updateAction({
+                title: notificationTitle,
+                metadata: {
+                  swapId: swap.id,
+                },
+                status: ActionStatus.Complete,
+                assetIds: [tradeSellAsset.assetId, tradeBuyAsset.assetId],
+              }),
+            )
+            store.dispatch(
+              swapSlice.actions.updateSwap({
+                id: swap.id,
+                status: SwapStatus.Success,
+                metadata: {
+                  ...swap.metadata,
+                  txLink,
+                },
+              }),
+            )
+
+            // @TODO: do we want to show this even if the user is on the trade confirmation page?
+            toast({
+              title: notificationTitle,
+              status: 'success',
+            })
+            return
+          }
+        }
+      } catch (error) {
+        console.error('Failed to fetch transaction details:', error)
+
+        store.dispatch(
+          actionCenterSlice.actions.updateAction({
+            metadata: {
+              swapId: swap.id,
+            },
+            status: ActionStatus.Complete,
+            assetIds: [tradeSellAsset.assetId, tradeBuyAsset.assetId],
+          }),
+        )
+        store.dispatch(
+          swapSlice.actions.updateSwap({
+            id: swap.id,
+            status: SwapStatus.Success,
+          }),
+        )
+
+        const notificationTitle = selectActionBySwapId(store.getState(), {
+          swapId: swap.id,
+        })?.title
+
+        // @TODO: do we want to show this even if the user is on the trade confirmation page?
+        toast({
+          title: notificationTitle,
+          status: 'success',
+        })
+
+        return
+      }
+    }
```

Comment:
This is not supposed to happen but exists as a security, and not blocking anything but real amounts, one improvement would be to flag the swap as "real amount not fetched" and so the action center could know that he needs to poll again the status to update the real amount